app/services/bulk_upload/sales/year2022/row_parser.rb

```diff
@@ -524,18 +528,18 @@ def attributes_for_log
 
     attributes["ethnic_group"] = ethnic_group_from_ethnic
     attributes["ethnic"] = field_30
```

Comment:
Do we perhaps want to also set `ethnic` to be nil if the buyer hasn't been interviewed?

Reply:
I had previously done this, but it gets derived anyway in the main log derivations so rather than setting it twice I'm leaving it as is